src/components/Icon/__stories__/Name.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import styled from '@emotion/styled'
+import { ComponentStory } from '@storybook/react'
+import Icon, { icons } from '..'
+import Stack from '../../Stack'
+
+const StyledStack = styled(Stack)`
+  font-size: ${({ theme }) => theme.typography.heading.fontSize};
+`
+
+export const Name: ComponentStory<typeof Icon> = args => (
+  <>
+    {icons.map(name => (
+      <StyledStack key={name} direction="row" gap={2} alignItems="center">
+        <Icon name={name} {...args} /> {name}
+      </StyledStack>
+    ))}
+  </>
+)
```

Comment:
it is not very beautiful, can't we just have icons with stack as parent?

<img width="335" alt="Screenshot 2022-11-08 at 14 18 09" src="https://user-images.githubusercontent.com/15812968/200574935-62000837-9b35-4c50-aad5-f9a1f7e7f655.png">